webapp/src/Form/Type/ContestType.php

```diff
@@ -77,6 +88,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'Yes' => true,
                 'No' => false,
             ],
+            'help' => 'When true, any logged in team can join this contest, and the teams/categories listed below are added by default. When false, only the teams/categories listed below will participate.',
```

Comment:
Can someone confirm this assumption is indeed correct?

Reply:
Nope when true the other fields are hidden (or should be) and everyone automatically is part of the contest. When false only the teams / cats you select are part. We don’t have a way to join as a team

Reply:
Ah right. They are hidden indeed.